### PR TITLE
Relval matrix updates

### PR DESCRIPTION
**Pull request by fabozzi**
- added 2016 relvals
- same steps for all 2015 relvals
- updated gensim recycling for 2017 relvals

### CODE REVIEW

**Comment by fabozzi** [comment]
please test

**Comment by fabozzi** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Comment by mmusich** [comment]
@fabozzi, the discussion with @Martin-Grunewald at  https://github.com/cms-sw/cmssw/issues/13819 made me realise that due to the snapshot features these relvals will consume wrong conditions.

**Comment by mmusich** [comment]
@fabozzi @hengne @davidlange6 would it be possible to wait for a refresh of the GT snasphots before sending the RelVals production for this cycle? @franzoni

**Comment by fabozzi** [comment]
@mmusich @franzoni I think it is good to have a working 2016 relval infrastructure merged in the release. We will handshake with Alca before submitting relvals to be provided to validators

**Comment by davidlange6** [comment]
hopefully working includes also the GT:)

**Comment by mmusich** [comment]
@davidlange6 technically speaking already works, but yields wrong results. 
Incidentally when https://github.com/cms-sw/cmssw/pull/14820/ will be merge the problem is going to be mitigated, but I would definitely wait for the DPGs to udpate the 2016 IOVs (due in any case for the data re-reco) before releasing for validation.

**Comment by davidlange6** [comment]
we have a different meaning of “works” then… but good if at least the prompt conditions are already about read to merge

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -183,6 +183,22 @@
 steps['RunSinglePh2015D']={'INPUT':InputInfo(dataSet='/SinglePhoton/Run2015D-v1/RAW',label='sigPh2015D',events=100000,location='STD', ls=Run2015D)}
 steps['RunZeroBias2015D']={'INPUT':InputInfo(dataSet='/ZeroBias/Run2015D-v1/RAW',label='zb2015D',events=100000,location='STD',ib_block='38d4cab6-5d5f-11e5-824b-001e67ac06a0',ls=Run2015D)}
 
+#### run2 2016B ####
+# Run2016B, 25ns: 274160
+Run2016B=selectedLS([274160],l_json=data_json2016)
```
Comment (slava77) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: is it possible to pick a run with CTPPS included?
274094 LS>151 or 274199 full

**Comment by mmusich** [comment]
@davidlange6 https://github.com/cms-sw/cmssw/pull/14842 is a temporary solution to avoid crazy output in relvals, @franzoni and I will take care of removing the keys once the standard re-reco GT is fully up-to-date. Is this a reasonable solution?

**Comment by fabozzi** [comment]
@slava77 run 274199 contains more than double statistics w.r.t. what we usually run with data relvals. What about to run only on a subset of it? For instance, the first 250 lumisections?

**Comment by slava77** [comment]
On 6/10/16 8:33 AM, fabozzi wrote:
> @slava77 <https://github.com/slava77> run 274199 contains more than
> double statistics w.r.t. what we usually run with data relvals. What
> about to run only on a subset of it? For instance, the first 250
> lumisections?

A fraction of it is fine.
I just wanted to make sure that we exercise the CTPPS software with
realistic inputs in the matrix.

Once this settles, can you please add one 2016 workflow to the short matrix?

Thanks

**Comment by fabozzi** [comment]
Il 10/06/16 17:50, Slava Krutelyov ha scritto:
> On 6/10/16 8:33 AM, fabozzi wrote:
>> @slava77 <https://github.com/slava77> run 274199 contains more than
>> double statistics w.r.t. what we usually run with data relvals. What
>> about to run only on a subset of it? For instance, the first 250
>> lumisections?
>
> A fraction of it is fine.
> I just wanted to make sure that we exercise the CTPPS software with
> realistic inputs in the matrix.
>
> Once this settles, can you please add one 2016 workflow to the short matrix?

Yes, sure

	Francesco

**Pull request merged**